examples/package.json

```diff
@@ -7,12 +7,12 @@
   },
   "main": "src/Application.container",
   "dependencies": {
-    "prop-types": "15.6.1",
-    "react": "16.8.6",
-    "react-native": "0.60.0",
+    "prop-types": "15.7.2",
+    "react": "17.0.2",
+    "react-native": "0.65.1",
     "react-native-fingerprint-scanner": "file:../"
   },
   "devDependencies": {
-    "metro-react-native-babel-preset": "^0.54.1"
+    "metro-react-native-babel-preset": "0.66.0"
```

Comment:
I believe the babel preset and RN version should be coincident. Any reason not to use RN latest? 0.66.4 I think right now (or .3 or .5 I forget...)

Reply:
Chosen 0.65.1 only because the discussed [issue](https://github.com/hieuvp/react-native-fingerprint-scanner/issues/192) has been related to that version.
I can use the latest. I'll update the PR late today.
About the babel-preset, I usually use the one shown in [react-native-upgrade-helper](https://react-native-community.github.io/upgrade-helper)